direct/types.py

```diff
@@ -5,12 +5,14 @@
 
 import pathlib
 from enum import Enum
-from typing import NewType, Union
+from typing import Dict, NewType, Union
 
```

Comment:
`Dict -> dict`. Also add `from __future__ import annotations` as the first line so you can do `A | B` rather than `Union[A, B]`